src/EpicsClient/EpicsClientFactory.cpp

```diff
@@ -8,8 +8,7 @@
 namespace Forwarder {
 namespace EpicsClient {
 
-using std::mutex;
-using ulock = std::unique_lock<mutex>;
+using ulock = std::lock_guard<std::mutex>;
```

Comment:
I think it would be better to avoid the `using` statement as it is only used twice and the name doesn't make much sense when it is no longer a `unique` lock.